eo-runtime/src/main/eo/org/eolang/collections/list.eo

```diff
@@ -95,7 +95,29 @@
         &.f x > @
 
   # For each array element dataize the object
-  [f] > each /bool
+  # Here "f" must be an abstract object with
+  # one free attribute, the element of the
+  # array.
+  [f] > each
+    if. > @
+      ^.is-empty
+      FALSE
+      seq
+        memory 0 > idx
+        while.
+          lt.
+            idx
+            arr.length
+          [i]
+            if. > @
+              eq.
+                (arr.at i).as-bytes
```

Comment:
@l3r8yJ I think this check is just checking the dataizated object equality to 0. Why do we need this?